index.html

```diff
@@ -64,14 +64,14 @@ <h2>I'm a Software Developer</h2>
             </div>
         </section>
         <section id="portafolio">
-            <div class="worksCards">
+            <div id="cardsContainer" class="worksCards">
                 <div id="WorkDiv" class="worksInit">
                     <div class="WorkTitle">
                         <h2>My Recent Works</h2>
                         <hr class="WorkSeparation">
                     </div>
-                    <div class="centerWork">
-                        <div class="WorkImage">
+                    <div id="centerWork" class="centerWork">
+                        <!--<div class="WorkImage">
```

Comment:
- [ ] I kindly suggest that you please remove commented out code from your work as it doesn't make your code neat.